tools/release

```diff
@@ -40,9 +47,11 @@ fi
 require_clean_work_tree 'build and upload a release'
 
 # Check the commit message
-commit_msg=$(git rev-list --format=%B --max-count=1 HEAD | tail -n +2)
-[ "$commit_msg" == "Release Zulip Server $version." ] \
-    || fail "Expected commit message: Release Zulip Server $version."
+if [ -z "$is_prerelease" ]; then
```

Comment:
I think release candidates use the same commit message format; so why do we need this conditional?

Reply:
I don't see a `6.0-rc1` commit on `main`?

Reply:
Yeah, it's on a separate branch that's pushed to `upstream`.